homeassistant/components/blink/camera.py

```diff
@@ -35,10 +35,22 @@ def __init__(self, data, name, camera):
         super().__init__()
         self.data = data
         self._attr_name = f"{DOMAIN} {name}"
+        self._camera_name = name
         self._camera = camera
         self._attr_unique_id = f"{camera.serial}-camera"
         _LOGGER.debug("Initialized blink camera %s", self.name)
 
+    @property
+    def device_info(self):
```

Comment:
As above, please use dataclass and attr format: `self._attr_device_info = DeviceInfo(...)`